conf/ref_databases.config

```diff
@@ -76,6 +60,30 @@ params {
             fmtscript = "taxref_reformat_gtdb.sh"
             dbversion = "GTDB R05-RS95 (https://data.ace.uq.edu.au/public/gtdb/data/releases/release95/95.0/)"
         }
+        'midori2-co1' {
+            title = "MIDORI2 - CO1 Taxonomy Database - Release GB250"
+            file = [ "http://reference-midori.info/download/Databases/GenBank250/DADA2_sp/uniq/MIDORI2_UNIQ_NUC_SP_GB250_CO1_DADA2.fasta.gz" ]
+            citation = "Machida RJ, Leray M, Ho SL, Knowlton N. Metazoan mitochondrial gene sequence reference datasets for taxonomic assignment of environmental samples. Sci Data. 2017 Mar 14;4:170027. doi: 10.1038/sdata.2017.27. PMID: 28291235; PMCID: PMC5349245."
+            fmtscript = "taxref_reformat_midori2.sh"
+            dbversion = "MIDORI2-CO1 GB250 (http://reference-midori.info/download/Databases/GenBank250/DADA2_sp/uniq/MIDORI2_UNIQ_NUC_SP_GB250_CO1_DADA2.fasta.gz)"
+            taxlevels = "Phylum,Class,Order,Family,Genus,Species"
+        }
+        'midori2-co1=gb250' {
+            title = "MIDORI2 - CO1 Taxonomy Database - Release GB250"
+            file = [ "http://reference-midori.info/download/Databases/GenBank250/DADA2_sp/uniq/MIDORI2_UNIQ_NUC_SP_GB250_CO1_DADA2.fasta.gz" ]
+            citation = "Machida RJ, Leray M, Ho SL, Knowlton N. Metazoan mitochondrial gene sequence reference datasets for taxonomic assignment of environmental samples. Sci Data. 2017 Mar 14;4:170027. doi: 10.1038/sdata.2017.27. PMID: 28291235; PMCID: PMC5349245."
+            fmtscript = "taxref_reformat_midori2.sh"
+            dbversion = "MIDORI2-CO1 GB250 (http://reference-midori.info/download/Databases/GenBank250/DADA2_sp/uniq/MIDORI2_UNIQ_NUC_SP_GB250_CO1_DADA2.fasta.gz)"
+            taxlevels = "Phylum,Class,Order,Family,Genus,Species"
+        }
+        'phytoref' {
+            title = "PhytoRef plastid 16S rRNA database for photosynthetic eukaryotes"
+            file = [ "http://phytoref.sb-roscoff.fr/static/downloads/PhytoRef_with_taxonomy.fasta" ]
+            citation = "Decelle, Johan, Sarah Romac, Rowena F. Stern, El Mahdi Bendif, Adriana Zingone, Stéphane Audic, Michael D. Guiry, et al. 2015. PhytoREF: A Reference Database of the Plastidial 16S rRNA Gene of Photosynthetic Eukaryotes with Curated Taxonomy. Molecular Ecology Resources 15 (6): 1435–45. https://doi.org/10.1111/1755-0998.12401."
+            fmtscript = "taxref_reformat_phytoref.sh"
+            dbversion = "unknown"
```

Comment:
They do not do versioning?
Would it be good to save the fasta or formatted database in the outdir folder so that is available afterwards as well? Is that needed?

Reply:
As far as I can see, it's not versioned. For the moment, I think we can keep it as it is and wait for comments.
I'll update to "unversioned".